aws_lambda_powertools/utilities/data_classes/bedrock_agent_event.py

```diff
@@ -77,10 +77,6 @@ def action_group(self) -> str:
     def api_path(self) -> str:
         return self["apiPath"]
 
-    @property
```

Comment:
Please add a comment saying that `httpMethod` is inherited from `BaseProxyEvent` class.